bedrock/firefox/templates/firefox/dev-whatsnew.html

```diff
@@ -43,7 +39,24 @@
       <p>{{_('Get to know the features that make it the most complete browser for building the Web.')}}</p>
     </header>
     <ul class="features">
-    {% if l10n_has_tag('spring_release') %}
+    {% if l10n_has_tag('firefox_dev_fall') and waffle.switch('firefox-dev-fall') %}
+      <li class="feature">
+        <a href="https://hacks.mozilla.org/2015/11/developer-edition-44-creative-tools-and-more" rel="external">
```

Comment:
This link is currently 404. That's probably intentional as the article is not ready, but just commenting for record.